improver_tests/acceptance/SHA256SUMS

```diff
@@ -633,73 +633,73 @@ e653ae271dd789dde8b03e6237cc7b48552dd9d5206d3430ee68d283683247a0  ./wind_downsca
+350f0754f7bb247a0a81bb81d05f3e69da4d8d768d8269c82f6fef73713b5b06  ./wxcode-modal/gridded_ties/20201209T1200Z-weather_symbols-PT01H.nc
+e8dc643ec6446d495a96d4dda6104b958d08137c15c3e832bb397a50229062eb  ./wxcode-modal/gridded_ties/20201209T1300Z-weather_symbols-PT01H.nc
+77375be1e8258f0b641ea6475c7739beec64252dc66ae4216a5cc02a3f01cd6e  ./wxcode-modal/gridded_ties/20201209T1400Z-weather_symbols-PT01H.nc
+148cdbc9282a8fb44e4793144852757ee36c5cd63d6efebfcb63337186e96a2f  ./wxcode-modal/gridded_ties/20201209T1500Z-weather_symbols-PT01H.nc
+44c510792f566329597239006b43ea736b5faa1a884202fd523aba31ebc4f445  ./wxcode-modal/gridded_ties/20201209T1600Z-weather_symbols-PT01H.nc
+182211dba9b8d18c2d48dce89f2871854ae9baa9457bc9cfad16c1cea80faf09  ./wxcode-modal/gridded_ties/20201209T1700Z-weather_symbols-PT01H.nc
+2ed494728e73e8f740680ae090504770ee6b60131d543698c640db5ed4e08598  ./wxcode-modal/gridded_ties/20201209T1800Z-weather_symbols-PT01H.nc
+fab89f125fe7b016a35fcbf52cd9b713cfa28c7009b90150f289f47d7bea6b12  ./wxcode-modal/gridded_ties/kgo.nc
+9fbcfc0e0c3f6af0d07690c6be12e2f641123168af0c7e165b8e0ec6a2ca6f98  ./wxcode-modal/single_input/20201210T0000Z-weather_symbols-PT01H.nc
+4e52a42de445064a987da6f93b7e888c0a8ca974063a4d9cf9818a57d9b60367  ./wxcode-modal/single_input/kgo.nc
+6513b722e1fdca6ad68acd768f3506ec814f78e9ccd4799972ec00a2316ef7cf  ./wxcode-modal/spot_input/20201209T0700Z-weather_symbols-PT01H.nc
+d4c5d626ad2e5a805f90f2902f6da5d33651eb6fdcb1e16116e2290d0c608bf6  ./wxcode-modal/spot_input/20201209T0800Z-weather_symbols-PT01H.nc
+4642fc43faf4bdde6a678578868fdebb22163064fe6ef0eef41f2f39a959ca89  ./wxcode-modal/spot_input/20201209T0900Z-weather_symbols-PT01H.nc
+ce6a854694aecd515063c91774eeb6160f39c9dbf5e970f72f960a3527e157e9  ./wxcode-modal/spot_input/20201209T1000Z-weather_symbols-PT01H.nc
+300c38e7f522ff1e9257708b183148f3441bb1a861c15eaff2a5f1713bbbc872  ./wxcode-modal/spot_input/20201209T1100Z-weather_symbols-PT01H.nc
+48ef16ad1fd0b499f1d38f585ddb0a4299c39e7d7df625ad0fa6c3dc24e0f094  ./wxcode-modal/spot_input/20201209T1200Z-weather_symbols-PT01H.nc
+5ab6b249ec615ddab9995629aaddbd85bba352ad52b7bcb0be5b3037d75c22ca  ./wxcode-modal/spot_input/20201209T1300Z-weather_symbols-PT01H.nc
+340205e12625ff9466c1afba8c007343aa40e21b837ab2f2143fa032e6c47bd3  ./wxcode-modal/spot_input/20201209T1400Z-weather_symbols-PT01H.nc
+776739d288b6e43b3d9341ba301feead160986d2540514f0663f243ca003e55a  ./wxcode-modal/spot_input/20201209T1500Z-weather_symbols-PT01H.nc
+4655358c22abdd0acfd5f4e08f156a7f4f629a1764dd8314f28f121964725078  ./wxcode-modal/spot_input/20201209T1600Z-weather_symbols-PT01H.nc
+28ee3f9d419bb9766cc68e1c74fa9e8e5010dc63a600635a54356a33795f6644  ./wxcode-modal/spot_input/20201209T1700Z-weather_symbols-PT01H.nc
+073fd0175e35afcf7f9fe61fedfa63bbd9ea5299f68d8b0287bc30085339b942  ./wxcode-modal/spot_input/20201209T1800Z-weather_symbols-PT01H.nc
+2041f9ed3d1c350518dab5d537652165e58bbe92a003f5c2dfaa0ff94ccce1e1  ./wxcode-modal/spot_input/kgo.nc
```

Comment:
In this file, it seems like the units for the `wmo_id` and `met_office_site_id` has changed from `unknown` to `1`. I think that the units should actually be `no_unit`.